user-interface/src/main/java/life/qbic/datamanager/views/general/footer/FooterComponent.java

```diff
@@ -23,13 +25,21 @@ public class FooterComponent extends Footer {
   private final RouterLink legalNoticeLink = new RouterLink("LegalNotice", LegalNotice.class);
   private final RouterLink dataPrivacyAgreement = new RouterLink("Data Privacy Agreement",
       DataPrivacyAgreement.class);
-  private final Anchor sourceCodeLink = new Anchor(
-      "https://github.com/qbicsoftware/data-manager-app", "Source", AnchorTarget.BLANK);
 
-  public FooterComponent() {
+  @Autowired
+  public FooterComponent(
+      @Value("${qbic.communication.data-manager.source-code.url}") String sourceCodeUrl,
+      @Value("${qbic.communication.documentation.url}") String documentationUrl,
+      @Value("${qbic.communication.api.url}") String apiUrl,
+      @Value("${qbic.communication.contact.email}") String contactEmail,
+      @Value("${qbic.communication.contact.subject}") String contactSubject) {
     setId("data-manager-footer");
     add(new Anchor(dataPrivacyAgreement.getHref(), "Data Privacy Agreement", AnchorTarget.BLANK),
         new Anchor(legalNoticeLink.getHref(), "Legal Notice", AnchorTarget.BLANK),
-        new Anchor(sourceCodeLink.getHref(), "Source", AnchorTarget.BLANK));
+        new Anchor(documentationUrl, "Documentation", AnchorTarget.BLANK),
+        new Anchor(apiUrl, "API", AnchorTarget.BLANK),
+        new Anchor(sourceCodeUrl, "Source", AnchorTarget.BLANK),
```

Comment:
Please remove the quotation mark from the default example in the application.properties. 
Otherwise the source link will be broken (Since vaadin cannot recognize the protocol and therefore assumes a relative link instead of an absolute link) 
![Screenshot 2024-06-14 at 13 10 41](https://github.com/qbicsoftware/data-manager-app/assets/29627977/3a0b02df-0416-4e3a-aef6-d64dccb6d3d7)